src/main/java/utils/command/AddTagToDeckCommand.java

```diff
@@ -42,6 +43,8 @@ private void addTagToDeck(DeckList deckList, TagList tagList, UserInterface ui)
             ui.printDeckCreationSuccess();
             deckToAdd = new Deck(deckName, tagUUID);
             deckList.addDeck(deckToAdd);
+        } else if(deckToAdd.tagInDeck(tagUUID)) {
```

Comment:
Same issue about function name

Reply:
Issues resolved in edab2bc2b42c131e9ab63358f987ffae5538ebed